addons/l10n_cl/models/account_move.py

```diff
@@ -114,7 +114,7 @@ def _get_last_sequence_domain(self, relaxed=False):
         if self.company_id.country_id.code == "CL" and self.l10n_latam_use_documents:
             where_string = where_string.replace('journal_id = %(journal_id)s AND', '')
```

Comment:
I think this replace is not needed -> here the inherit method

![image](https://user-images.githubusercontent.com/52895221/102730007-5433b500-4333-11eb-89e9-a6f9a68a4166.png)

Reply:
Not error, my mistake!